.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -28,27 +28,41 @@ Terraform v0.13.3
 ```
 
 ### Configuration File(s)
-```
-Copy config HCL here
-```
+_Reminder to redact any sensitive information that may be present in this file_
+
+<details>
```

Comment:
Didn't know about this feature. Nice. :+1:

Reply:
@-findkim found this 🎉 !